cli/ops/timers.rs

```diff
@@ -77,7 +81,7 @@ pub fn init(rt: &mut deno_core::JsRuntime) {
   super::reg_json_sync(rt, "op_global_timer_stop", op_global_timer_stop);
   super::reg_json_sync(rt, "op_global_timer_start", op_global_timer_start);
   super::reg_json_async(rt, "op_global_timer", op_global_timer);
-  super::reg_json_sync(rt, "op_now", op_now);
+  rt.register_op("op_now", metrics_op(minimal_op(op_now)));
```

Comment:
This is debatable, maybe we shouldn't take metrics of `op_now`?

Reply:
The wonders of copy-pasta
Will fix

Reply:
@Soremwar this is ok, I'm just wondering if we add additional overhead to the `op_now`. Maybe let's keep it for now and address it if other performance problems occur